<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MySettingsDialog</class>
 <widget class="QWidget" name="MySettingsDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>520</width>
    <height>380</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>520</width>
    <height>380</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>520</width>
    <height>380</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QLabel" name="SettingsTitle">
   <property name="geometry">
    <rect>
     <x>170</x>
     <y>0</y>
     <width>180</width>
     <height>60</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>14</pointsize>
     <bold>true</bold>
    </font>
   </property>
   <property name="text">
    <string>        SETTINGS</string>
   </property>
  </widget>
  <widget class="QGroupBox" name="RoundTimeBox">
   <property name="geometry">
    <rect>
     <x>50</x>
     <y>60</y>
     <width>200</width>
     <height>30</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>14</pointsize>
    </font>
   </property>
   <property name="title">
    <string>RoundTime</string>
   </property>
   <widget class="QSpinBox" name="RoundTimeSpin">
    <property name="geometry">
     <rect>
      <x>120</x>
      <y>0</y>
      <width>80</width>
      <height>30</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>11</pointsize>
     </font>
    </property>
    <property name="minimum">
     <number>10</number>
    </property>
    <property name="maximum">
     <number>90</number>
    </property>
    <property name="singleStep">
     <number>5</number>
    </property>
    <property name="value">
     <number>30</number>
    </property>
   </widget>
  </widget>
  <widget class="QGroupBox" name="GameTimeBox">
   <property name="geometry">
    <rect>
     <x>50</x>
     <y>110</y>
     <width>400</width>
     <height>30</height>
    </rect>
   </property>
   <property name="minimumSize">
    <size>
     <width>90</width>
     <height>20</height>
    </size>
   </property>
   <property name="font">
    <font>
     <pointsize>14</pointsize>
    </font>
   </property>
   <property name="title">
    <string>GameTime</string>
   </property>
   <widget class="QSpinBox" name="GameTimeH">
    <property name="geometry">
     <rect>
      <x>120</x>
      <y>0</y>
      <width>80</width>
      <height>30</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>11</pointsize>
     </font>
    </property>
    <property name="maximum">
     <number>3</number>
    </property>
   </widget>
   <widget class="QSpinBox" name="GameTimeMin">
    <property name="geometry">
     <rect>
      <x>230</x>
      <y>0</y>
      <width>80</width>
      <height>30</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>11</pointsize>
     </font>
    </property>
    <property name="minimum">
     <number>0</number>
    </property>
    <property name="maximum">
     <number>59</number>
    </property>
    <property name="value">
     <number>40</number>
    </property>
   </widget>
   <widget class="QLabel" name="HLabel">
    <property name="geometry">
     <rect>
      <x>205</x>
      <y>4</y>
      <width>21</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>h</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_3">
    <property name="geometry">
     <rect>
      <x>316</x>
      <y>5</y>
      <width>54</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>min</string>
    </property>
   </widget>
  </widget>
  <widget class="QPushButton" name="EscSettings">
   <property name="geometry">
    <rect>
     <x>460</x>
     <y>20</y>
     <width>31</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>ESC</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
